### PR TITLE
Fixed symfony 3 compatibility issues (routing & form-type)

### PR DESCRIPTION
**Pull request by chirimoya**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | yes
| BC breaks? | yes
| Deprecations? | no
| Fixed tickets | part of #2106 
| Related issues/PRs | sulu/sulu-document-manager#101
| License | MIT
| Documentation PR | none

#### What's in this PR?

Routing & form-type issues.

#### Why?

Symfony 3 compatibility.

### CODE REVIEW

**Comment by wachterjohannes** [comment]
@chirimoya remove the form.type aliases from tags

File: src/Sulu/Bundle/ContentBundle/Form/Type/PageDocumentType.php [hunk 1]
```diff
@@ -63,15 +63,15 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     /**
      * {@inheritdoc}
      */
-    public function setDefaultOptions(OptionsResolverInterface $options)
+    public function configureOptions(OptionsResolver $options)
```
Comment (wachterjohannes) [inline src/Sulu/Bundle/ContentBundle/Form/Type/PageDocumentType.php]: do we use any other form-types? can you check if there are other calls like this?

File: src/Sulu/Bundle/SnippetBundle/DependencyInjection/SuluSnippetExtension.php [hunk 2]
```diff
@@ -70,7 +71,7 @@ public function prepend(ContainerBuilder $container)
                 'sulu_document_manager',
                 [
                     'mapping' => [
-                        'snippet' => ['class' => SnippetDocument::class, 'phpcr_type' => 'sulu:snippet'],
+                        'snippet' => ['class' => SnippetDocument::class, 'phpcr_type' => 'sulu:snippet', 'form_type' => SnippetType::class],
```
Comment (wachterjohannes) [inline src/Sulu/Bundle/SnippetBundle/DependencyInjection/SuluSnippetExtension.php]: i would refactor the SnippetController to use the metadata to get the form-type

File: src/Sulu/Bundle/ContentBundle/Form/Type/PageDocumentType.php (hunk 1 above)
Comment (chirimoya) [inline src/Sulu/Bundle/ContentBundle/Form/Type/PageDocumentType.php]: fixed

File: src/Sulu/Bundle/SnippetBundle/DependencyInjection/SuluSnippetExtension.php (hunk 2 above)
Comment (chirimoya) [inline src/Sulu/Bundle/SnippetBundle/DependencyInjection/SuluSnippetExtension.php]: fixed

**Pull request merged**